<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmEncriptar</class>
 <widget class="QMainWindow" name="FrmEncriptar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Encriptar archivo o mensaje</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnEncriptar">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>80</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Encriptar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>encriptar.png</normaloff>encriptar.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLimpiar">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>140</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Limpiar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>clean.png</normaloff>clean.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCargar">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>200</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cargar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>upload.png</normaloff>upload.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>471</width>
      <height>511</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Texto">
     <attribute name="icon">
      <iconset>
       <normaloff>../../../../.designer/backup/text.png</normaloff>../../../../.designer/backup/text.png</iconset>
     </attribute>
     <attribute name="title">
      <string>Texto</string>
     </attribute>
     <widget class="QTextEdit" name="textMensaje">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>451</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lblMensaje">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>451</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Mensaje</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblMensaje_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>451</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Mensaje encriptado</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblMensajeEncriptado">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>451</width>
        <height>181</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Imagen">
     <attribute name="icon">
      <iconset>
       <normaloff>../../../../.designer/backup/picture.png</normaloff>../../../../.designer/backup/picture.png</iconset>
     </attribute>
     <attribute name="title">
      <string>Imagen</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>431</width>
        <height>281</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../../.designer/backup/picture.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lblMensaje_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>451</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Imagen</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnDescargar">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>260</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Descargar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>download.png</normaloff>download.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEnviar">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>320</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enviar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>send.png</normaloff>send.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEncriptar">
    <property name="title">
     <string>Encriptar</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuLimpiar">
    <property name="title">
     <string>Limpiar</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuCargar">
    <property name="title">
     <string>Cargar</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSalir">
    <property name="title">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
   </widget>
   <addaction name="menuEncriptar"/>
   <addaction name="menuLimpiar"/>
   <addaction name="menuCargar"/>
   <addaction name="menuSalir"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
